Dockerfile

```diff
@@ -1,12 +1,14 @@
-FROM golang:1.5.3-onbuild
+FROM golang:1.5.4-alpine
 
-# update debian packages
-RUN apt-get update
-
-# install uchiwa-web bower package
-RUN apt-get install -yq nodejs npm git wget
-RUN ln -s /usr/bin/nodejs /usr/bin/node
-RUN npm install --production --unsafe-perm
+# golang alpine doesn't have ONBUILD, do it manually, then run npm and cleanup
+COPY . /go/src/app
+WORKDIR /go/src/app
+RUN apk add --no-cache nodejs git && \
+    go get -d -v && \
```

Comment:
This requires setting `GOVENDOREXPERIMENT` since you're still on  go1.5.
Upgrading to go1.6 already has this set by default.